src/Spryker/Zed/AppMerchant/AppMerchantConfig.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * This file is part of the Spryker Suite.
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace Spryker\Zed\AppMerchant;
+
+use Spryker\Glue\AppMerchantBackendApi\Plugin\GlueApplication\MerchantAppOnboardingBackendApiRouteProviderPlugin;
+use Spryker\Shared\AppKernel\AppKernelConstants;
+use Spryker\Shared\GlueJsonApiConvention\GlueJsonApiConventionConstants;
+use Spryker\Zed\Kernel\AbstractBundleConfig;
+
+class AppMerchantConfig extends AbstractBundleConfig
+{
+    /**
+     * @api
+     *
+     * @var string
+     */
+    public const MERCHANT_ONBOARDING_STATUS = 'merchant-onboarding-status';
+
+    public function getAppMerchantAppOnboardingApiUrl(): string
+    {
+        return $this->getStringValue(GlueJsonApiConventionConstants::GLUE_DOMAIN) . MerchantAppOnboardingBackendApiRouteProviderPlugin::ROUTE_URL_PATH;
+    }
+
+    public function getAppIdentifier(): string
+    {
+        return $this->getStringValue(AppKernelConstants::APP_IDENTIFIER);
+    }
+
+    public function getOnboardingType(): string
+    {
+        return 'implement on project level';
+    }
+
+    protected function getStringValue(string $configKey): string
+    {
+        /**
+         * @api
+ *
+    * @phpstan-var string */
```

Comment:
broken formatting

Reply:
FIxed. Now I see what the issue with the @api tag is. It's sometimes there and sometimes not. I see it appearing when there is som doc block like for arrays then CS will add the @apip tag when there is no block nothing will be added.

Should be checked later at some point. For me not a blocker.